WeatherProbe/Patches/WeatherRegistryPatches.cs

```diff
@@ -0,0 +1,14 @@
+namespace WeatherProbe{
+
+	internal class WeatherRegistryPatches{
+
+		public static bool IsModPresent => BepInEx.Bootstrap.Chainloader.PluginInfos.ContainsKey("mrov.WeatherRegistry");
+
+		public static void SetWeatherOnHost(SelectableLevel level, LevelWeatherType weatherType){
+			if(IsModPresent){
```

Comment:
Whenever you call this function, you're already forced to have the types used in this method loaded so the check is useless inside the method definition.